sass/styles/chart.scss

```diff
@@ -285,14 +285,75 @@ html, body {
 
 
 /* ------------------------------------------------------- CHART OVERLAY ELEMENTS ------------------------------------------------------ */
+#mSticky{
```

Comment:
this is for [the design](https://app.zeplin.io/project/5b3d8a662f4775474e13e26f/screen/5b431ab44db03a8165123a82) of removing Comparison by right click.